### PR TITLE
fix(useResponsive): use isBrowser instead of windowExists

### PR DESCRIPTION
**Pull request by ihaichao**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    fix(useResponsive): use isBrowser instead of windowExists       |
| 🇨🇳 Chinese |    useResponsive: 使用 isBrowser 替换 windowExists       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useResponsive/index.ts [hunk 1]
```diff
@@ -49,8 +50,9 @@ export function configResponsive(config: ResponsiveConfig) {
 }
 
 export function useResponsive() {
-  const windowExists = typeof window !== 'undefined';
-  if (windowExists && !listening) {
+  if (!isBrowser) return {};
```
Comment (crazylxr) [inline packages/hooks/src/useResponsive/index.ts]: 这里为啥返回的是 空对象呢，以前的是 undefined？

Comment (ihaichao) [reply packages/hooks/src/useResponsive/index.ts]: 噢没注意，已修改

**Review by crazylxr — APPROVED** [review]

File: packages/hooks/src/useResponsive/index.ts [hunk 2]
```diff
@@ -49,8 +50,9 @@ export function configResponsive(config: ResponsiveConfig) {
 }
 
 export function useResponsive() {
```
Comment (brickspert) [inline packages/hooks/src/useResponsive/index.ts:52]: 这里有问题的。

以前在没有 window 的情况下，返回的是 info，现在返回的是 `undefined`，很不合理。

之前的更合理。

Comment (ihaichao) [reply packages/hooks/src/useResponsive/index.ts:52]: 没有 window 的情况下， info 就是 `undefined` 啊

**Pull request merged**